.github/workflows/build.yml

```diff
@@ -9,4 +9,30 @@ jobs:
     name: Run Pester tests
     steps:
     - uses: actions/checkout@v2
-    - run: pwsh -f ./module/run-tests.ps1
\ No newline at end of file
+    - run: pwsh -f ./module/run-tests.ps1
```

Comment:
does that have to be hard coded - can it be a variable / glob etc?

Reply:
I'm not sure how - it already points to the single file that discovers and runs all the tests

Reply:
If it means we can reuse without alteration, then that's great.